prow/gerrit/adapter/adapter.go

```diff
@@ -238,6 +238,7 @@ func (c *Controller) Sync() {
 		}()
 
 		changes := c.gc.QueryChangesForInstance(instance, syncTime, c.config().Gerrit.RateLimit)
+		timeCompletedGettingChanges := time.Now()
```

Comment:
why not reuse `now`?

Reply:
We need a timestamp *after* the query completes to measure the time it takes for us to process the changes.